string text
int line
---
string status
